spec/controllers/idv/review_controller_spec.rb

```diff
@@ -528,6 +528,7 @@ def show
               expect(response).to redirect_to idv_personal_key_path
 
               enrollment.reload
+
```

Comment:
I liked this being separated, but this is also just a TK for adding the test. Wanted to get something rough up today, but I still need to add a test for reproofing.